hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStoreIterator.java

```diff
@@ -52,6 +53,16 @@ public RDBStoreIterator(ManagedRocksIterator iterator, RDBTable table) {
     this(iterator, table, null);
   }
 
+  public RDBStoreIterator(
+      ManagedRocksIterator iterator, RWBatchOperation rwBatch,
+      RDBTable table) throws IOException {
```

Comment:
The write batch iterator should probably be its own subclass of `RDBStoreIterator`, since the `rwBatch` field is not used when the class is used as a normal Rocks DB iterator.

Reply:
Updated...